deployment/deployment-grpc-v2/02-contour.yaml

```diff
@@ -50,8 +56,6 @@ spec:
         command: ["contour"]
         args:
         - bootstrap
-        # Uncomment the statsd-enable to enable prometheus metrics
-        #- --statsd-enable
         # Uncomment to set a custom stats emission address and port
         #- --stats-address=0.0.0.0
         #- --stats-port=8002
```

Comment:
@stevesloka should we change the commented out stats-port to something besides 8002?